tests/swagger_generation/test_swagger_generator.py

```diff
@@ -157,15 +157,15 @@ class FooSchema(m.Schema):
             uid = m.fields.String()
             name = m.fields.String()
 
-        class ListOfFooSchema(m.Schema):
+        class NestedFoosSchema(m.Schema):
             data = m.fields.Nested(FooSchema, many=True)
 
         class FooUpdateSchema(m.Schema):
             __swagger_title = 'FooUpdate'
 
             name = m.fields.String()
 
-        class FooListSchema(m.Schema):
+        class NameAndOtherSchema(m.Schema):
             name = m.fields.String()
             other = m.fields.String()
```

Comment:
Maybe this should have a better name than NameAndOtherSchema, but at least that's less confusing than calling it FooListSchema.

Reply:
Haha yeah, no perfect names here.